snippets/customers/login-form.liquid

```diff
@@ -1,52 +1,52 @@
-                    <input class="form-control" ng-model="password" type="password" name="customer[password]" required vc-flying-placeholder="{{ 'customer.sign_in.password' | t }}" />
-                    <span class="text-danger" ng-show="!customer_login['customer[password]'].$pristine && customer_login['customer[password]'].$error.required">{{ 'common.required' | t }}</span>
-
+                <div class="col-md-12">
+                    <div class="form-group">
+                        <input class="form-control" ng-model="password" type="password" name="customer[password]" required vc-flying-placeholder="{{ 'customer.sign_in.password' | t }}" />
+                        <span class="text-danger" ng-show="!customer_login['customer[password]'].$pristine && customer_login['customer[password]'].$error.required">{{ 'common.required' | t }}</span>
+                    </div>
                 </div>
             </div>
         </div>
-    </div>
-
-    {% if template != 'customers/login' %}
-    <div class="form-group">
-        {% include 'customers/forgot-links' %}
-    </div>
-    {% endif %}
 
-    {% if template != 'customers/login' %}
-    <div class="row">
-        <div class="col-md-5">
-    {% else %}
-            <div class="form-group">
-    {% endif %}
-                 <input type="submit" class="btn btn-primary btn-block" value="{{ 'customer.sign_in.sign_in' | t }}">
         {% if template != 'customers/login' %}
-            </div>
-        {% endif %}
-        {% if template != 'customers/login' %}
-            <div class="col-md-5">
-                {{ 'layout.customer.register' | t | customer_register_link | replace: 'a href', 'a class="btn btn-alt btn-primary" href' }}
-            </div>
-        {% endif %}
-        </div>
-        {% if template == 'customers/login' %}
         <div class="form-group">
             {% include 'customers/forgot-links' %}
         </div>
         {% endif %}
-</fieldset>
-{{ antiforgery }}
+
+        {% if template != 'customers/login' %}
+        <div class="row">
+            <div class="col-md-5">
+        {% else %}
+                <div class="form-group">
+        {% endif %}
+                     <input type="submit" class="btn btn-primary btn-block" value="{{ 'customer.sign_in.sign_in' | t }}">
+            {% if template != 'customers/login' %}
+                </div>
+            {% endif %}
+            {% if template != 'customers/login' %}
+                <div class="col-md-5">
+                    {{ 'layout.customer.register' | t | customer_register_link | replace: 'a href', 'a class="btn btn-alt btn-primary" href' }}
+                </div>
+            {% endif %}
+            </div>
+            {% if template == 'customers/login' %}
+            <div class="form-group">
+                {% include 'customers/forgot-links' %}
+            </div>
+            {% endif %}
+    </fieldset>
+    <input name="RememberMe" type="hidden" value="true" />
```

Comment:
Move to localization

Reply:
That's not display name, it's `input` name, i.e. field in model in storefront. So don't need to fix anything